go.mod

```
@@ -1,6 +1,6 @@
 module github.com/hashicorp/vault-plugin-auth-kubernetes
 
-go 1.25.3
+go 1.24.9
```

I don't think we need to be too strict about the patch version for this directive, since it's just a minimum version. Setting it to "go 1.24" and running `go mod tidy` results in `1.24.0` so we might want to stick with that.

```suggestion
go 1.24.0
```